app/src/main/java/app/musikus/ui/activesession/ActiveSessionViewModel.kt

```diff
@@ -428,28 +362,7 @@ class ActiveSessionViewModel @Inject constructor(
         }
     }
 
-    private fun itemClicked(item: LibraryItem) {
-        applicationScope.launch {
-            try {
-                activeSessionUseCases.selectItem(item)
-            } catch (e: IllegalStateException) {
-                Log.e(LOG_TAG, "Cannot start new section: ${e.message}")
-            }
-        }
-    }
-
-    private fun togglePause() {
-        applicationScope.launch {
-            if (activeSessionUseCases.getPausedState()) {
-                activeSessionUseCases.resume()
-            } else {
-                activeSessionUseCases.pause()
-            }
-        }
-    }
-
     private fun stopSession() {
-        val endDialogData = _endDialogData.value ?: return
         applicationScope.launch {
```

Comment:
Can you remind me again why this is inside the applicationScope? 😅

Reply:
`getFinalizedSession` is suspend
`sessionUseCases.add` is suspend

Reply:
Yes, but why not use the ViewModel scope?